src/components/Book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Book = book => (
+  <tr>
+    <td>{book.id}</td>
+    <td>{book.name}</td>
+    <td>{book.category}</td>
+  </tr>
+);
+
```

Comment:
[Optional]
You should practise the usage of `prop-types` in all the necessary files, for some reason you're not having error on linter about it, magic right? 😄 

https://reactjs.org/docs/typechecking-with-proptypes.html

You could have something like 👇

```
Book.propTypes = {
  book: PropTypes.instanceOf(Array).isRequired,
};
```

Reply:
> you're not having error on linter about it, magic right?

:rofl: